### PR TITLE
Make db connection sslmode verify-full

### PR DESCRIPTION
**Pull request by anthonyhashemi**
## Changes in this PR

- Make db connection sslmode verify-full

## JIRA ticket

https://national-archives.atlassian.net/browse/AYR-1024

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,6 +232,8 @@ The database is assumed to be a PostgreSQL database.
 
 This could be spun up by PostgreSQL or from Amazon RDS, for example.
 
+One thing to note is that our database connection's ssl mode is `verify-full` therefore your db instance must have a root certificate on its server, and you must have a copy of a leaf leaf certificate signed by the root certificate in a location accessible by this flask webapp, specified by `DB_SSL_ROOT_CERTIFICATE`.
```
Comment (colinbowen) [inline README.md]: PR looks good, just a minor change to remove the duplicate word "leaf" from the README

Comment (colinbowen) [reply README.md]:
```suggestion
One thing to note is that our database connection's ssl mode is `verify-full` therefore your db instance must have a root certificate on its server, and you must have a copy of a leaf certificate signed by the root certificate in a location accessible by this flask webapp, specified by `DB_SSL_ROOT_CERTIFICATE`.
```

**Review by colinbowen — DISMISSED** [review]
Approving just a minor text change needed.

**Comment by anthonyhashemi** [comment]
> Approving just a minor text change needed.

Fixed 👌

**Review by colinbowen — APPROVED** [review]

**Pull request merged**